frontend/containers/open-call-application-modal/component.tsx

```diff
@@ -0,0 +1,210 @@
+                  id: 'b6bipu',
+                })}
+              />
+            </div>
+            <Combobox
+              control={control}
+              controlOptions={{ disabled: false }}
+              aria-required
+              name="project_id"
+              id="project-id"
+              className="mt-2.5 w-full h-10 border border-beige rounded-lg px-4"
+              height="small"
+              placeholder={formatMessage({
+                defaultMessage: 'select project',
+                id: 'kRnyPA',
+              })}
+              aria-describedby="project-id-error"
+            >
+              {sortedProjects.map(({ id, name }) => (
+                <Option key={id}>{name}</Option>
+              ))}
+            </Combobox>
+            <ErrorMessage id="message-error" errorText={errors?.project_id?.message} />
+
+            <span className="mt-2">
+              <Link href={Paths.ProjectCreation} passHref>
+                <a className="inline text-sm text-gray-600 underline rounded-full focus-visible:outline focus-visible:outline-green-dark focus-visible:outline-2 focus-visible:outline-offset-2">
+                  <FormattedMessage
+                    defaultMessage="Don’t have a project yet? Create a project now"
+                    id="SFCYg4"
+                  />
+                </a>
+              </Link>
+            </span>
+
+            <div className="mt-4">
+              <Label className="mr-2" htmlFor="message">
+                <FormattedMessage defaultMessage="Type your message" id="HtMY9H" />
+              </Label>
+              <FieldInfo
+                content={formatMessage({
+                  defaultMessage: 'Tell the investor why your project should receive the funding.',
+                  id: '3WLO/M',
+                })}
+              />
+            </div>
+            <Textarea
+              id="message"
+              name="message"
+              register={register}
+              aria-describedby="message-error"
+            />
+            <ErrorMessage id="message-error" errorText={errors?.message?.message} />
+          </>
+        )}
+        {alert && (
+          <Alert type="warning" className="my-4 -mb-4 rounded">
+            {/* useGetAlert returns an array, but the endpoint only sends one error message at a time. */}
+            {alert[0]}
+          </Alert>
+        )}
+
+        <div className="flex justify-end mt-8">
+          <Button
+            theme="secondary-green"
+            size="small"
+            className="flex-shrink-0 mr-5"
+            onClick={closeModal}
+          >
+            <FormattedMessage defaultMessage="Cancel" id="47FYwb" />
+          </Button>
+          <Button
+            className="flex-shrink-0 mr-5"
+            theme="primary-green"
+            size="small"
+            disabled={isSubmitting || isLoadingProjects}
+            onClick={handleSubmit(onSubmit)}
+          >
```

Comment:
This should be a submit button:
```suggestion
          <Button
            type="submit"
            className="flex-shrink-0 mr-5"
            theme="primary-green"
            size="small"
            disabled={isSubmitting || isLoadingProjects}
            onClick={handleSubmit(onSubmit)}
          >
```